network/transparent_endpointclient_linux.go

```diff
@@ -189,12 +189,5 @@ func (client *TransparentEndpointClient) ConfigureContainerInterfacesAndRoutes(e
 }
 
 func (client *TransparentEndpointClient) DeleteEndpoints(ep *endpoint) error {
```

Comment:
should we log indicating skipping endpoint deletion? It will help debugging any issues with transparent mode.

Reply:
imo, i dont think its necessary because we skip deleting interface in all cases. If we are not deleting based on some condition then logs will be useful